### ISSUE TITLE
Tags with multiple words not showing up in search

### ISSUE THREAD

**Issue by todd-tech-writer** [squidfunk/mkdocs-material#4004]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

When I add a tag with multiple words in it on one of my markdown pages, the page does not show up when searching for the tag. 

### Expected behaviour

I'd expect when searching for a multiple-word tag on my website, for the page with the tag to appear at the top of the search results.

### Actual behaviour

I added the tags `Correct Tag` and `Find Me` to `test-file.md`. When I search for the tag `Find Me` in the search bar of my website, the `test-file.md` page is not shown in the search results. When I only type `Find` in the search bar, it will populate. See the screenshots below for an example:

![image](https://user-images.githubusercontent.com/79108878/172483471-bada8cc8-4522-4350-bd94-9b91f3af3d13.png)

![image](https://user-images.githubusercontent.com/79108878/172483507-6a8bd7c3-adcf-485a-8871-e64e583d23d6.png)

![image](https://user-images.githubusercontent.com/79108878/172483524-ac6ed0b5-0ea2-4848-8bc8-1d520c435a94.png)

### Steps to reproduce

1. Create a simple `mkdocs.yml` with tags and metadata
2. Create a tag on `test-file.md` called `Find Me`
3. Run mkdocs serve
4. Search for `Find Me`

### Package versions

- Python: `3.8`
- MkDocs: `1.3.0`
- Material: `8.2.8`

### Configuration

```yaml
site_name: 'Test Site'
site_description: 'Description'
site_author: 'Company'

nav:
  - Home:
    - Introduction: index.md
    - Getting Started: test-file.md

theme:
  name: 'material'

markdown_extensions:
  - meta

plugins:
  - search
  - tags
```

### System information

- Operating system: macOS Monterey 12.4
- Browser: Chrome

**Comment by todd-tech-writer** [comment]
Awesome, thank you!

**Comment by squidfunk** [comment]
The fix was just released as part of 8.3.4.
